README.md

```diff
@@ -1,6 +1,7 @@
-# okdata-token-service
+Token Service
+======================
```

Comment:
Synes egentlig den gamle stilen ser bedre ut, jeg. :sweat_smile: Mer konsistent med de andre nivåene (`##`, `###`, osv), og tar mindre plass.

Reply:
Gikk for ==== fordi den så ut til å være mest brukt i andre repoer